aspnetcore/release-notes/aspnetcore-9/includes/delete_keys.md

```diff
@@ -0,0 +1,9 @@
+<!--
+[!INCLUDE[](~/release-notes/aspnetcore-9/includes/delete_keys.md)]
+-->
+
+### Data Protection support for deleting keys
+
+Previous to .NET 9, data protection keys were ___not___ deletable by design, to prevent data loss. Deleting a key renders its protected data inaccessible. Given their small size, the accumulation of these keys generally posed minimal impact. However, to accommodate extremely long-running services, we have introduced the option to delete keys. Generally, only old keys should be deleted. Only delete keys when you can accept the risk of data loss in exchange for storage savings. We recommend data protection keys should ___not___ be deleted.
```

Comment:
```suggestion
Prior to .NET 9, data protection keys were ___not___ deletable by design, to prevent data loss. Deleting a key renders its protected data irretrievable. Given their small size, the accumulation of these keys generally posed minimal impact. However, to accommodate extremely long-running services, we have introduced the option to delete keys. Generally, only old keys should be deleted. Only delete keys when you can accept the risk of data loss in exchange for storage savings. We recommend data protection keys should ___not___ be deleted.
```

I don't feel strongly about it.